### DISCUSSION TITLE
Delete old container complete

### DISCUSSION THREAD

**Discussion by rdebeuze** [fabianishere/udm-iptv#177] [General]
Hello,

In https://github.com/fabianishere/udm-iptv/discussions/143 we noticed i also run the old setup and new one. Fabian gave me 2 commands:

podman stop iptv
podman rm iptv

I noticed i have to run these after every software update or reboot. Can i remove the old container setup complete so i haven't to do that?

**Comment by fabianishere** [comment]
You need to remove the boot script from your device. Probably it's located at `/mnt/data/on_boot.d/15-iptv.sh`.

**Comment by rdebeuze** [comment]
Thank you did that and works!
